docs/rpc/tezos-rpc-documentation/getBlockHeader.md

```diff
@@ -0,0 +1,67 @@
+# getBlockHeader
+
+### How to use it 
+
+```typescript
+// Import required libraries and modules from Tatum SDK
+import { TatumSDK, Tezos, Network } from '@tatumio/tatum';
+
+// Initialize the Tatum SDK for Tezos
+const tatum = await TatumSDK.init<Tezos>({ network: Network.TEZOS });
+
+// Define the chain ID (For this example, we use "NetXdQprcVkpaWU" as a placeholder)
+const params = { 
+    chainId: 'NetXdQprcVkpaWU',
```

Comment:
Why one time you have const params = { 
    chainId: 'YOUR_CHAIN_ID',           // Specify the chain ID (Network identifier)
    blockId: 'YOUR_BLOCK_ID',           // Optional: Specify the block ID (hash)
};

And second time
const params = { 
    chainId: 'NetXdQprcVkpaWU',           // Specify the chain ID (Network identifier)
    blockId: 'BLVRfBFc8Mb7s4PUacDfaeQTPj9HtzTNEArM1Ni1YbxRuK73htp',           // Optional: Specify the block ID (hash)
};

?

Reply:
fixed